common/src/main/java/dev/schmarrn/lighty/ui/SettingsScreen.java

```diff
@@ -112,11 +112,11 @@ protected void init() {
     }
 
     @Override
-    public void render(@NotNull GuiGraphics guiGraphics, int i, int j, float f) {
+    public void render(@NotNull GuiGraphics guiGraphics, int mouseX, int mouseY, float delta) {
```

Comment:
I know that this is a good intended change, but please only change the things you need in your PR.  Changing unrelated things may make porting changes more difficult than it already is.

Reply:
Sorry about this one. It was a change I made after the pull request. I didn't know that it would be added to the same PR. 

I'll make a new PR for this.